opencog/nlp/chatbot/README

```diff
@@ -4,75 +4,30 @@
                 Linas Vepstas <linasvepstas@gmail.com>
                           August 2009
 
-XXX Important parts of the chatbot are being worked on, and so the
-chatbot might be broken in small or large ways.  I beleive that the
-following revisions should mostly work: 3150, 3470.
-XXX
-
 A quick-n-dirty hookup of the OpenCog NLP pipeline to an IRC chatbot.
 
 At this time, the bot is a demo of the OpenCog natural-language
 processing (NLP) pipeline; it is NOT -- repeat NOT -- a demo of
-OpenCog reasoning, deduction, or anything like that.  The chatbot
-is as dumb as a rock, as thick as a brick; it possesses no intelligence
-whatsoever.
-
-It can, however, answer simple English-language questions. It does so
-by comparing the syntactical structure of the question to previously
-entered sentences.  Thus, for example, if the chatbot is told that
-"John threw a rock", and then asked "Who threw a rock?", direct
-comparison allows it to equate "Who" -> "John".  The comparisons are
-done on the syntactic structure (dependency parse) of the sentence:
-that is, the parsed form of the sentences and questions are compared.
-This allows for some sophistication: the system can correctly answer
-"What did John throw?" for the above input, because dependency parsing
-will correctly identify "throw" as the head verb of the sentence.
-
-Since OpenCog can save its contents to a database, it can "remember"
-a large number of assertions that have been previously entered, and
-answer questions about those.  Currently, a pre-parsed copy of the
-MIT ConceptNet can be loaded into OpenCog.
-
-Please note: this is meant to be a DEMO of the NLP pipeline, and litle
-more. It is up to you to do something with it!  Also: it is subject to
-change at any time, as I am likely to twiddle, tweak and change it about.
-(Adding reasoning is a difficult but important priority).
+OpenCog reasoning, deduction, or anything like that.  
+
+To make the chatbot to actually answer the question with a correct answer
+PLN inference should be used and that would be the next implementation step.
+
```

Comment:
The chatbot should be able to answer many many questions without requiring any PLN inference whatsoever.  For example:  "The ball is under the table. Where is the ball?" requires no inference at all.